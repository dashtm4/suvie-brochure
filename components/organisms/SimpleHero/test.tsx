import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { SimpleHero } from '.'

describe('<SimpleHero />', () => {
  const children = 'Test children'
  const imageSrc = 'image.jpg'
  const mobileHeight = 500
  const desktopHeight = 700

  it('should renders correct <SimpleHero />', () => {
    const wrapper = shallow(
      <SimpleHero
        mobileHeight={mobileHeight}
        desktopHeight={desktopHeight}
        imageSrc={imageSrc}
      >
        {children}
      </SimpleHero>
    )

    expect(wrapper.contains(children)).toBeTruthy()

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
