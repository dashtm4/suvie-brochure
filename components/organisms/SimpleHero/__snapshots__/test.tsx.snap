// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SimpleHero /> should renders correct <SimpleHero /> 1`] = `
<div
  className="bg-no-repeat bg-cover bg-center flex flex-col items-center justify-center text-center"
  style={
    Object {
      "backgroundImage": "url(image.jpg)",
      "height": 500,
    }
  }
>
  Test children
</div>
`;
